User.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+namespace AirportTicketBooking
+{
+    internal class User
+    {
+        private static int _idCounter = 0;
+        public int Id { get; private set; } = GenerateUniqueID();
+        public string Name { get; set; }
+        public string Email { get; set; }
+
+        private static int GenerateUniqueID()
```

Comment:
is this about the user itself or about the type of id?
we can create a new directory called Utilities and add GuidUtilities right there to generate a random Guid